.github/workflows/main.yml

```diff
@@ -25,22 +25,21 @@ jobs:
     strategy:
       matrix:
         versions:
-        - rails: '5.2'
-          ruby: '2.7'
```

Comment:
ruby 2.7 reached its EOL

Reply:
Outside scope of this PR, but could be nice to make a dummy app for Ruby 3.X/Rails 7 and run tests against them in CI